### DISCUSSION TITLE
Alerts feature overwhelmed by "dust storm deposits"

### DISCUSSION THREAD

**Discussion by hutchwilco** [guydavis/machinaris#599] [Q&A] [answered]
Unsure if this is affecting many others, but due to the dust storm of "thousands of Cha-ching! Just received 0.000000000001 XCH" deposits in recent weeks, the Alerts page is getting bogged down trying to load ~15,000 entries, and sort them into pages of 10 per page.

Can I suggest a method be provided for users to go into the database and query that term and remove?

Thanks!

**Comment by guydavis** [comment, marked as answer]
Hi, have not seen that, but here's the SQL: 
![image](https://user-images.githubusercontent.com/12936/156900717-1a5ca4b9-89c1-4ce9-9a32-84a9f4eb1609.png)

**Reply by hutchwilco** [reply]
thank you! this removed ~14,500 entries from the dust storm deposits
